tools/slicec-cs/src/slicec_ext/enum_ext.rs

```diff
@@ -4,7 +4,7 @@ use crate::slicec_ext::primitive_ext::PrimitiveExt;
 use slicec::grammar::*;
 
 pub trait EnumExt {
-    fn get_underlying_cs_type(&self) -> String;
+    fn get_underlying_cs_type(&self) -> String; // TODO This could become 'str'!
```

Comment:
Might be better to open an issue to update this so it doesn't get lost or forgotten about.

Reply:
If you'd prefer it. It just felt pretty small to me, and I've already started implementing it : v)

Reply:
🔧

Reply:
https://github.com/icerpc/icerpc-csharp/issues/3926